api/v1alpha1/package.go

```diff
@@ -23,3 +29,24 @@ func (config *Package) GetValues() (values map[string]interface{}, err error) {
 	err = yaml.Unmarshal([]byte(config.Spec.Config), &mapInterfaces)
 	return mapInterfaces, err
 }
+
+func (config *Package) GetClusterName() string {
+	if strings.HasPrefix(config.Namespace, namespacePrefix) {
+		clusterName := strings.TrimPrefix(config.Namespace, namespacePrefix)
+		// Backward compatibility
+		if clusterName == oldPbcName {
+			return os.Getenv(clusterNameEnvVar)
```

Comment:
How exactly does this work?

Reply:
If the installation has the old PBC name which is something like `eksa-package-bundle-controller`, then use the environment variable to get the cluster name. I probably will follow up with a PR to remove support for this and some migration.